lib/index.coffee

```diff
@@ -13,6 +13,7 @@ Understream.mixin JSONParser, 'parseJSON'
 module.exports = (keys, options={}) ->
   _(options).defaults
     delim: '\t'
+    tmp_dir: null
```

Comment:
Mostly to make it clear this was an option since we don't really have docs. I can remove if we feel that's better.